src/pyxir/runtime/backends/vai_rt/vai_compute_func.hpp

```diff
@@ -18,12 +18,19 @@
 
 #include <memory>
 #include <unordered_set>
+
+#ifndef USE_VAI_RT_DPUCAHX8H
```

Comment:
Could you replace this with #if defined(USE_VAI_RT) || defined(USE_VAI_RT_DPUCZDX8G) and also include the vai_api/dpu_func.hpp here instead of below. And put USE_VAI_RT_DPUCAHX8H in an elif?